app/scripts/modules/amazon/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -17,7 +17,7 @@ module.exports = angular.module('spinnaker.aws.serverGroupCommandBuilder.service
 
     function buildNewServerGroupCommand (application, defaults) {
       defaults = defaults || {};
-      var regionsKeyedByAccountLoader = accountService.getRegionsKeyedByAccount('aws');
+      var regionsKeyedByAccountLoader = accountService.getCredentialsKeyedByAccount('aws');
```

Comment:
I hate to ask, but can you rename this variable to something that makes more sense, like `credentialsLoader`? Also change `regions` to `credentials` or something more accurate in the rest of this method.